docs/enhancement-proposals/off-ep-0003.md

```diff
@@ -0,0 +1,86 @@
+# OFF-EP 0003 — Allow interpolation of non-integer bond orders
+
+**Status:** Proposed
+
+**Authors:** Matt Thompson, matt.thompson@openforcefield.org
+
+**Stakeholders:** David Mobley, John Chodera, Jeffrey Wagner, Simon Boothroyd
+
+**Acceptance criteria:** Unanimity
+
+**Created:** 2021-11-23
+
+**Discussion:** [Issue #8](https://github.com/openforcefield/standards/issues/8)
+
+**Implementation:** [PR #23](https://github.com/openforcefield/standards/pull/23)
+
+## Abstract
+
+# OFF-EP 0003 — Allow interpolation of non-integer bond orders
+
+This OFF-EP explicitly documents that non-integer bond orders can be used for interpolating parameters.
+
+## Motivation and Scope
+
+The SMIRNOFF specification describes a process by which fractional bond orders can be used to
+interpolate parameters for
+[bond](https://openforcefield.github.io/standards/standards/smirnoff/#fractional-bond-orders) and
+[proper
+torsion](https://openforcefield.github.io/standards/standards/smirnoff/#fractional-torsion-bond-orders)
+forces. The examples provided use bond orders of 1 and 2 and as the basis of interpolation between
```

Comment:
"The examples provided use bond orders of 1 and 2 and as the basis of interpolation between their associated parameters."

This sentence is nonsensical. Was some text omitted between "and" and "as the basis"?

Reply:
What's missing here is jargon for "the x coordinates," like your above "fractional bond order interpolation points." There's also an added "and" that makes is hard to read.